packages/webapp/src/pages/ConnectionList.tsx

```diff
@@ -40,7 +62,89 @@ export default function ConnectionList() {
             setLoaded(true);
             getConnections();
         }
-    }, [getConnectionListAPI, loaded, setLoaded]);
+    }, [getConnectionListAPI, loaded, setLoaded, limit, offset, selectedIntegration, initialIntegration, initialOffset]);
+
+    useEffect(() => {
+        const getFilters = async () => {
+            if (connections.length > 0) {
+                const res = await fetch(`/api/v1/connection-filters/`, {
+                    method: 'GET',
+                    headers: {
+                        'Content-Type': 'application/json',
+                        Accept: 'application/json'
+                    }
+                });
+
+                if (res?.status === 200) {
+                    try {
+                        const filters = await res.json();
+
+                        if (filters) {
+                            if (filters.integrations.length > 0) {
```

Comment:
you can use a single condition with optional chaining `if (filters?.integrations.length > 0)`